src/RecursiveArrayTools.jl

```diff
@@ -9,6 +9,7 @@ using RecipesBase, StaticArraysCore, Statistics,
     ArrayInterface, LinearAlgebra
 using SymbolicIndexingInterface
 using SparseArrays
+import Static # so it isn't a stale dep, used in FastBroadcastExt
```

Comment:
oh wait, oh no, we really cannot use that here. That will destroy compile times.